bears/python/MypyBear.py

```diff
@@ -0,0 +1,75 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from coalib.bears.requirements.PipRequirement import PipRequirement
+
+OUT_RE = (r'(?P<filename>[^:]+):(?P<line>\d+)?: '
+          '(?P<severity>[^:]+): (?P<message>.*)')
+
+
+@linter(executable='mypy',
+        output_format='regex',
+        output_regex=OUT_RE,
+        severity_map={'error': RESULT_SEVERITY.NORMAL,
+                      'note': RESULT_SEVERITY.INFO},
+        )
+class MypyBear:
+    """
+    Type-checks your Python files!
+
+    Checks optionsl static typing using the mypy tool.
+    See <http://mypy.readthedocs.io/en/latest/basics.html> for info on how to
+    add static typing.
+
+    :param language:
+        Set to "Python" or "Python 3" to check Python 3.x source. Use
+        "Python 2" for Python 2.x.
+    :param python_version:
+        Set the specific Python version, e.g. "3.5".
+    :param silent_imports:
+        Don't follow imports to .py files.
+    :param check_untyped_defs:
+        Type check the interior of functions without type annotations.
+    :param disallow_untyped_defs:
+        Disallow defining functions without type annotations or with
+        incomplete type annotations.
+    :param disallow_untyped_calls:
+        Disallow calling functions without type annotations from functions.
+    :param strict_optional:
+        Enable experimental strict Optional checks.
```

Comment:
can you add this documentation to the function where it belongs? coala will merge the docs as well.